src/lkl/setup.c

```diff
@@ -1341,6 +1380,9 @@ void lkl_start_init()
 
     register_lkl_syscall_overrides();
 
+    sgxlkl_shared_memory_t* shm = &sgxlkl_enclave_state.shared_memory;
+    const sgxlkl_enclave_config_t* cfg = sgxlkl_enclave_state.config;
+
     // Provide LKL host ops and virtio block device ops
     lkl_host_ops = sgxlkl_host_ops;
 
```

Comment:
Yes, that's correct, they have to be in the whitelist to work. I haven't seen any test failures because of this yet, so I didn't touch it. The whole logging/tracing business will have to be a separate PR anyways.

Reply:
There are no test failures because there's no test that tests logging. But developers use these env vars quite frequently so we should make sure it's still working, otherwise you will get reports as soon as this PR is merged.

Reply:
That's fine then; they are relying on an insecure mechanism that we removed. We could try and check the env vars in the launcher and issues a warning if they try to use them.

Reply:
The best would be to add them to the enclave config and add the corresponding "overridable" env vars. Wouldn't that then just work?

Reply:
Yes it would, except that it would be debug-only. I think that's fine, but others may disagree. Since there will be a broader tracing + logging refactoring as a separate effort, I'm inclined to wait for that, but adding them to the schema/structs wouldn't be a lot of effort either.

Reply:
I think it's fine if it's debug and non-release only. That's what's used for debugging anyway.